presto-main/src/main/java/com/facebook/presto/execution/SqlQueryManager.java

```diff
@@ -286,6 +289,9 @@ public QueryInfo createQuery(Session session, String query)
         QueryExecution queryExecution;
         Statement statement;
         try {
+            if (query.length() > maxQueryLength) {
+                throw new PrestoException(EXCEEDED_QUERY_LENGTH_LIMIT, String.format("Maximum query length limit exceeded: %d", query.length()));
```

Comment:
Always use `%s` instead of `%d`. Only use `%d` with custom formatting like `%02d`.

How about this: `"Query text length (%s) exceeds the maximum length (%s)"`